Prox/Prox/Data/TravelTimesProvider.swift

```diff
@@ -45,10 +49,33 @@ struct TravelTimesProvider {
             completion(travelTime)
         }
     }
+
+    static func travelTime(fromLocation: CLLocationCoordinate2D, toLocation: CLLocationCoordinate2D, byTransitTypes transitTypes: [MKDirectionsTransportType], withCompletion completion: @escaping ((TravelTimes?) -> ())) {
+        var allTimes = [TravelTimes?]()
+        for transitType in transitTypes {
```

Comment:
Having all the travel time data keeps things simple when we need to choose what to display the user. 👍 

However, if we make these requests in quick succession, as we do here, would they rate limit us?